<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Nilai</class>
 <widget class="QDialog" name="Nilai">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Nilai</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_tb1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Tugas Besar 1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_tb2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Tugas Besar 2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_l1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Latihan 1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_l2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Latihan 2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_q1">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>380</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Quiz 2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_q2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>350</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Quiz 1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_uts">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>410</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>UTS</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_uas">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>440</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>UAS</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Tambah">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Tambah</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Update">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Delete">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Clear">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
